### ISSUE TITLE
Noise coefficient

### ISSUE THREAD

**Issue by ruiixu23** [mattja/sdeint#11]
I hope it's not a too stupid question since I have no background in SDEs in general but I need to use the package to generate sample paths for our machine learning project. 

From my understanding, suppose I have a set of SDEs with d states, if I assume independent additive constant diffusion noise, then the G function should return a dxd diagonal matrix right? Should the entries of the matrix contain the "variance or std" of the noise? I am not even sure if the terms variance and std are proper here. 

Thanks a lot for helping.

**Comment by mattja** [comment]
Yes, in that case the G function should return a constant dxd diagonal array.

Very roughly speaking, the entries correspond to std, not variance. More precisely, if function f were zero (no drift) then for a diagonal G the diffusion process would have its mean at the initial value and the variance in each direction increasing linearly with time, at a rate proportional to the square of those entries.
